src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceEducation.vue

```diff
@@ -193,12 +196,8 @@ export default defineComponent({
     numOfEducationRequired() {
       let numOfEducationRequired = 1;
 
-      this.wizardStore.wizardData[this.wizardStore.wizardConfig.steps.certificationType.form.inputs.certificationSelection.id]?.includes(
-        CertificationType.SNE,
-      ) && numOfEducationRequired++;
-      this.wizardStore.wizardData[this.wizardStore.wizardConfig.steps.certificationType.form.inputs.certificationSelection.id]?.includes(
-        CertificationType.ITE,
-      ) && numOfEducationRequired++;
+      this.applicationStore.draftApplication?.certificationTypes?.includes(CertificationType.SNE) && numOfEducationRequired++;
+      this.applicationStore.draftApplication?.certificationTypes?.includes(CertificationType.ITE) && numOfEducationRequired++;
```

Comment:
maybe a good getter function on application store?

Reply:
I think this is the only component where we use this function. So my thoughts are we don't need to put it into the store?